napari/layers/points/points.py

```diff
@@ -705,8 +705,8 @@ def _extent_data(self) -> np.ndarray:
         if len(self.data) == 0:
             extrema = np.full((2, self.ndim), np.nan)
         else:
-            maxs = np.max(self.data, axis=0)
-            mins = np.min(self.data, axis=0)
+            maxs = np.max(self.data, axis=0) + self._current_size
```

Comment:
```suggestion
            maxs = np.max(self.data + self.size, axis=0)
            mins = np.min(self.data - self.size, axis=0)
```

Reply:
this would take the full size into account but might be slow

Reply:
done